src/Operators/diffusion.jl

```diff
@@ -0,0 +1,26 @@
+####
+#### Diffusive fluxes
+####
+
+@inline diffusive_flux_x(i, j, k, grid, c, κ_fcc) = κ_fcc * AxF(i, j, k, grid) * δx_faa(i, j, k, grid, c)
+@inline diffusive_flux_y(i, j, k, grid, c, κ_cfc) = κ_cfc * AyF(i, j, k, grid) * δy_afa(i, j, k, grid, c)
+@inline diffusive_flux_z(i, j, k, grid, c, κ_ccf) = κ_ccf *  Az(i, j, k, grid) * δz_aaf(i, j, k, grid, c)
+
+####
+#### Laplacian diffusion operator
+####
+
+"""
+    ∇κ∇c(i, j, k, grid, c, κ)
+
+Calculates diffusion for a tracer c via
+
+    1/V * [δx_caa(κ * Ax * δx_faa(c)) + δy_aca(κ * Ay * δy_afa(c)) + δz_aac(κ * Az * δz_aaf(c))]
+
+which will end up at the location `ccc`.
+"""
+@inline function ∇κ∇c(i, j, k, grid, c, κ)
```

Comment:
Is this function used anywhere? Each `TurbulenceClosure` implements a specific diffusive flux divergence; it may not make sense to generalize. This function isn't valid for tensor diffusivities anyways.

Reply:
As an aside, its nice when the function preserves the semantics of the original math; eg the function `∇κ∇c` might have argument order `κ, c`. This kind of function also typically will need to dispatch on the type of `κ`, thus by julia convention `κ` would come sooner.

Reply:
Good point about the order. Otherwise this probably belongs in the `TurbulenceClosures` submodule. See my comment above.